superset/assets/stylesheets/guavus.less

```diff
@@ -0,0 +1,66 @@
+//=== Inverted navbar
+// Reset inverted navbar basics
+@navbar-inverse-bg:                         #25373e;
+@navbar-inverse-link-color:                 #fff;
+@navbar-inverse-link-hover-color:           #fff;
+@navbar-height:                    45px;
+@navbar-inverse-brand-hover-color: @navbar-inverse-link-color;
+
+.headerImgDiv {
```

Comment:
use kebaba-case with BEM convention or adhere to superset convention

Reply:
done